gridt/src/app/settings/change-password/change-password.page.ts

```diff
@@ -47,7 +47,7 @@ export class ChangePasswordPage {
 
     el.present();
 
-    this.api.postPassword$(form.value.old_password, form.value.new_password).pipe(timeout(500)).subscribe(
+    this.api.changePassword$(form.value.old_password, form.value.new_password).pipe(timeout(500)).subscribe(
```

Comment:
Here a timeout of half a second remains, is there a reason for that?

Reply:
Nope, will remove in next commit.